src/main/resources/tags/fluid_sensitive_materials.json

```diff
@@ -0,0 +1,7 @@
+{
+	"values" : [
```

Comment:
Not sure but shouldn't buttons and redstone conponents be included too? As well as cake, pressure plates...

Reply:
Not really